### PR TITLE
release notes for 1.30

### PR DESCRIPTION
**Pull request by PtitBen**

### CODE REVIEW

File: docs/community/releases.md [hunk 1]
```diff
@@ -6,6 +6,46 @@ description: Here are our release notes - enjoy!
...
 
+**Progress 👣**
+
+*"We can only see a short distance ahead, but we can see plenty there that needs to be done."*
+This is what Alan Turing said about making progress and knowing what is left to achieve. We thought it was a great way to introduce our brand new `ns-progress` component. Showing progress and setting expectations is a great incentive to complete a complex task. This is what we want to enable with this release. That, and bug fixes, improvements and further work on design tokens!
+
+Release date - 2021/05/06
+
+**New features 🏁**
+* `ns-progress` gives enables you to show progress throughout a task. Wether it's short and easy or long and complex, you can set an expectation bny showing how many steps there are, where a user stands, and what's left and coming next. The component is very flexible and easy to setup, all the way to its completed state! [Learn more about it here](https://nucleus.design/docs/components/ns-progress) or [give it a go in Storybook](https://www.britishgas.co.uk/nucleus/demo/index.html?path=/story/components-ns-progress--standard).
```
Comment (RobTobias123) [inline docs/community/releases.md:17]: remove 'gives' first sentence as you have 'enables' or vice versa.

**Review by dbenmore — APPROVED** [review]

File: docs/community/releases.md [hunk 2]
```diff
@@ -6,6 +6,46 @@ description: Here are our release notes - enjoy!
...
+
+**Improvements 🖋**
+* `ns-datepicker` - We improved the focus on the inputter when inline validation triggers an error for a missing date and focuses back on the datepicker
+* `ns-datepicker` - We moved the dialog outside of shadowroot, for improved displaying in context of other components.
```
Comment (RobTobias123) [inline docs/community/releases.md:21]: US vs English spelling?  Dialog/Dialogue

Comment (RobTobias123) [reply docs/community/releases.md:21]: perhaps change 'improved displaying' for 'display improvement'?

File: docs/community/releases.md [hunk 3]
```diff
@@ -6,6 +6,46 @@ description: Here are our release notes - enjoy!
 
 ## Version 1.29.0
```
Comment (RobTobias123) [inline docs/community/releases.md]: v1.30 ?

Comment (PtitBen) [reply docs/community/releases.md]: That should be fixed with the second commit :)

File: docs/community/releases.md (hunk 2 above)
Comment (PtitBen) [reply docs/community/releases.md:21]: I think as it refers to the "dialog" element, it needs the american spelling ?

File: docs/community/releases.md (hunk 1 above)
Comment (PtitBen) [reply docs/community/releases.md:17]: Argh .... good spot !!!

**Pull request merged**